tabrepo/simulation/ensemble_selection_config_scorer.py

```diff
@@ -339,48 +384,46 @@ def from_repo(cls, repo: "EvaluationRepository", **kwargs):
             **kwargs,
         )
 
-    def evaluate_task(self, task: str, models: List[str]) -> Tuple[float, np.array]:
+    def evaluate_task(self, task: str, models: List[str]) -> dict[str, ...]:
         tid, fold = task_to_tid_fold(task=task)
         dataset = self.tid_to_dataset_name_dict[tid]
         return self.ensemble_scorer.evaluate_task(dataset=dataset, fold=fold, models=models)
 
-    def compute_errors(self, configs: List[str]) -> Tuple[Dict[str, float], Dict[str, np.array]]:
+    def compute_errors(self, configs: list[str]) -> dict[str, dict[str, ...]]:
```

Comment:
The ellipsis is probably unintended? If you mean to indicate a type too complex, this would be better as it would be the right type:
```suggestion
    def compute_errors(self, configs: list[str]) -> dict[str, dict[str, object]]:
```

For cases with complex outputs, consider using a dataclass as it is generally recommended over dict (it has a lot of advantages)

Reply:
Agreed object makes more sense. Regarding dataclass, I generally agree but it would be a non-trivial lift to refactor I think. Will consider this later on for TabRepo 2.0.

Reply:
for some reason git isn't allowing me to commit the suggestion directly, so I sent a commit separately